scripts/build/Platform/Windows/build_config.json

```diff
@@ -449,12 +449,24 @@
         "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo"
     }
   },
+  "install_release": {
+    "TAGS": [],
+    "COMMAND": "build_windows.cmd",
+    "PARAMETERS": {
+      "CONFIGURATION": "release",
+      "OUTPUT_DIRECTORY": "build\\windows_mono",
+      "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0 -DLY_MONOLITHIC_GAME=TRUE -DLY_DISABLE_TEST_MODULES=TRUE",
```

Comment:
Should that Generator stay pinned to VS2019?

Reply:
oh interesting,  I just went with what we had but if it's OK to use the highest e.g. VS 2022 then I'm down for that, just didn't want to rock the boat too hard.